transports/wasm-ext/src/websockets.js

```diff
@@ -80,7 +80,18 @@ const dial = (addr) => {
 			read: (function*() { while(ws.readyState == 1) { yield reader.next(); } })(),
 			write: (data) => {
 				if (ws.readyState == 1) {
-					ws.send(data);
+          // The passed in `data` is an `ArrayBufferView` [0]. If the
```

Comment:
This file seems to be tab-indented but the new code here is using spaces which I think is why this is rendering as un-aligned?